src/payment/strategies/ppsdk/ppsdk-sub-strategy.ts

```diff
@@ -8,5 +9,9 @@ export interface SubStrategySettings {
 }
 
 export interface SubStrategy {
-    execute(settings: SubStrategySettings): Promise<void>;
+    execute(settings: SubStrategySettings): void;
+
+    initialize(options?: PaymentInitializeOptions): void;
```

Comment:
Likewise, you're waiting for this method to complete: https://github.com/bigcommerce/checkout-sdk-js/blob/9123b21a61f2b9d69d38a6bbed99d6dd19c3f962/src/payment/strategies/ppsdk/ppsdk-strategy.ts#L104